api/src/main/java/uk/gov/hmcts/fees/register/api/controllers/ChargeableFeeWrapperDto.java

```diff
@@ -1,26 +1,20 @@
 package uk.gov.hmcts.fees.register.api.controllers;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonUnwrapped;
-import lombok.Builder;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import uk.gov.hmcts.fees.register.legacymodel.Fee;
 
 @Data
+@AllArgsConstructor
+@NoArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class ChargeableFeeWrapperDto<F extends Fee> {
     @JsonUnwrapped
-    private final F fee;
+    private F fee;
 
-    private final int chargeableFee;
+    private int chargeableFee;
 
-    @JsonCreator
```

Comment:
@JsonUnwrapped field combination is not supported with @JsonCreator by jackson-databind.